ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -4,12 +4,164 @@
 
 declaration template components/${project.artifactId}/schema;
 
-include 'quattor/schema';
+include 'quattor/types/component';
+include 'components/accounts/functions';
+
+# TODO: some generic types, candidates for template-library-core
+@documentation{
+    hwloc location, e.g. node:1 for NUMAnode 1
```

Comment:
Expand the abbreviation - hardware locality - and perhaps reference hwloc(7) to make it clear this is not a physical location.